### ISSUE TITLE
"rdf:type" sometimes incorrectly replaced by "a"

### ISSUE THREAD

**Issue by janvlug** [atextor/owl-cli#12]
If you apply `owl-cli write` on this ontology:
```
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix myont: <https://myontology/> .
@prefix sh: <http://www.w3.org/ns/shacl#> .

myont:MyClass-type rdf:type sh:PropertyShape ;
  sh:path rdf:type .
```
the results is:
```
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix myont: <https://myontology/> .
@prefix sh: <http://www.w3.org/ns/shacl#> .

myont:MyClass-type a sh:PropertyShape ;
  sh:path a .
```
The resulting last line `sh:path a .` looks weird (maybe even incorrect, I'm not sure, but some software fails on it).

I would expect that `rdf:type` in the last line was not replaced by `a`. So the line should stay as it was: `sh:path rdf:type .`.

**Comment by atextor** [comment]
Hi @janvlug,
you are correct, this is a bug. The `a` syntax is only allowed when `rdf:type` is used as a predicate, not when it's used as an object.

**Comment by atextor** [comment]
This is now fixed in [1.2.5](https://github.com/atextor/owl-cli/releases/tag/v1.2.5).
